src/masonite/orm/providers/InternalORMProvider.py

```diff
@@ -0,0 +1,13 @@
+from ...providers.Provider import Provider
+
+from ...commands import MakeModelCommand
+
+
+class InternalORMProvider(Provider):
+    """Allow to tweak official ORMPRovider when used inside Masonite."""
+
+    def __init__(self, application):
+        self.application = application
+
+    def register(self):
+        self.application.make("commands").swap(MakeModelCommand(self.application))
```

Comment:
Then in here we do 

```python
MakeModelCommand(self.application, default_model_path=model_path())
```

Reply:
Yep this would avoid duplication, I will try